int32 robot_id
int32 robot_type
int32 currNode
bool isCurrVisited
int32 nextNode
bool isNextVisited
float32 height
